task/transcode.go

```diff
@@ -161,6 +161,13 @@ func TaskTranscode(tctx *TaskContext) (*data.TaskOutput, error) {
 	outFiles = append(outFiles, mp4muxer)
 	outBuffers = append(outBuffers, ws)
 	var transcoded [][]byte
+	contentResolution := ""
+	for _, track := range tctx.InputAsset.AssetSpec.VideoSpec.Tracks {
+		if track.Type == "video" {
+			contentResolution = fmt.Sprintf("%dx%d", track.Width, track.Height)
+			break
+		}
+	}
```

Comment:
**nit:** We should probably create a litle helper/utility to grab the video track from an asset spec (or maybe receiving a `trackType string` and returning the first track with that). We have this exact same loop repeated in a couple of places now.